playbooks/installer/ospd/undercloud/pre.yml

```diff
@@ -72,3 +72,9 @@
       lineinfile:
           dest: "/etc/sudoers"
           line: "{{ installer.user.name }} ALL=(root) NOPASSWD:ALL"
+
+- name: Deploy and configure external loadbalancer instance
+  hosts: loadbalancer
+  roles:
+    - { role: "loadbalancer/provision/", when: groups.loadbalancer is defined }
```

Comment:
remove "provision", leave it as:
`- { role: "loadbalancer", when: groups.loadbalancer is defined }`